libraries/ms-drivers/src/ltc_afe_impl.c

```diff
@@ -211,13 +244,11 @@ StatusCode ltc_afe_impl_trigger_cell_conv(LtcAfeStorage *afe) {
 }
 
 StatusCode ltc_afe_impl_trigger_aux_conv(LtcAfeStorage *afe, uint8_t device_cell) {
-  // TODO(SOFT-9): Update GPIO usage to match updated design
-
-  // configure the mux to read from cell
-  // we use GPIO2, GPIO3, GPIO4, GPIO5 to select which input to read
-  // corresponding to the binary representation of the cell
-  prv_write_config(afe, (device_cell << 4) | LTC6811_GPIO1_PD_OFF);
-
+  uint8_t gpio_bits =
+      LTC6811_GPIO1_PD_OFF | LTC6811_GPIO3_PD_OFF | LTC6811_GPIO4_PD_OFF | LTC6811_GPIO5_PD_OFF;
+  prv_write_config(afe, (device_cell << 4) | gpio_bits);
```

Comment:
You should remove the `device_cell << 4` which was there to toggle GPIO2-5 based on the cell to read